uspv/init.go

```diff
@@ -13,67 +13,48 @@ import (
 	"github.com/mit-dci/lit/lnutil"
 )
 
-// Connect dials out and connects to full nodes.
-func (s *SPVCon) Connect(remoteNode string) error {
+func (s *SPVCon) GetSeedAdrs(seed string) ([]string, error) {
```

Comment:
as it no longer uses SPVCon, it doesn't need to be a method

Reply:
nice catch, fixing.